app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="sections">
+    <% unless user.photo.nil? %>
+        <%= image_tag (user.photo), height: '115', width: '115' %>
+    <% end %>
+    <div class="users">
+        <h1><%= link_to (user.name), user%></h1>
+        <span>Number of posts: <%= user.posts_counter %></span>    
+    </div>
+</div>

```

Comment:
- [ ] Great work for using partial view :100: :+1:

Reply:
Changes implemented on the READMe